### DISCUSSION TITLE
Endpoint lat/lon not showing

### DISCUSSION THREAD

**Discussion by SiscoMajada** [GitSquared/edex-ui#971] [Questions] [answered]
On the eDEX-UI WORLD VIEW the endpoint Lat/Lon shows as "Unknown"
Is there a fix for this or is it always showing unknown. Network traffic is working just fine but world view shows online and after a few minutes it shows offline.
How do i get it to show Lat/Lon ?
Thanks in advance for your answers/help
Update:
After moving my pc to where my router is the lat/lon pop up immediately. When moving back to my hobby room it does not show. Coincidence? I don't know but it always work when im close to the router.
![offline hobbyroom](https://user-images.githubusercontent.com/70756197/102897930-13d65300-4469-11eb-8432-50bbf45f482f.jpg)
![kitchen](https://user-images.githubusercontent.com/70756197/102897943-1769da00-4469-11eb-98f4-4b9c3fcc6e19.jpg)

**Comment by GitSquared** [comment]
You might be having an issue with the geoip databases being corrupted after an update. Try deleting [config dir](https://github.com/GitSquared/edex-ui/wiki/userData)/geoIPcache and starting edex again. It'll download new databases automatically, watch for a spike in the network monitor. Once it has finished, the world view will be updated will all the relevant info.

**Reply by alex-engelmann** [reply]
Having same issue, tried deleting the geoIPcache directory and it redownloaded but still not seeing LAT/LON.  

IPv4 address is shown as --.--.--.-- too, not sure if related.  Ping and network traffic are working though.

**Reply by GitSquared** [reply]
If there's a geoIPcache-tmp folder, you might want to try deleting that too. Check you're on the latest version, too - v2.2.5 released this morning.

**Reply by alex-engelmann** [reply]
Yeah I deleted the tmp directory and am using the latest version

**Comment by GitSquared** [comment, marked as answer]
I'll be tracking this in #964 from now on, thanks for reporting
